src/sentry/api/endpoints/release_thresholds/release_threshold_details.py

```diff
@@ -27,8 +27,8 @@
 class ReleaseThresholdPUTSerializer(serializers.Serializer):
     threshold_type = serializers.ChoiceField(choices=ReleaseThresholdType.as_str_choices())
     trigger_type = serializers.ChoiceField(choices=ReleaseThresholdTriggerType.as_str_choices())
-    value = serializers.IntegerField()
-    window_in_seconds = serializers.IntegerField(min_value=0)
+    value = serializers.IntegerField(required=True, min_value=0)
+    serializers.IntegerField(required=True, min_value=0)
```

Comment:
I think the field got deleted by accident
```suggestion
    window_in_seconds = serializers.IntegerField(required=True, min_value=0)
```

Reply:
Ah oops thank you